RecoHGCal/TICL/plugins/PatternRecognitionbyCA.cc

```diff
@@ -306,6 +306,27 @@ void PatternRecognitionbyCA<TILES>::mergeTrackstersTRK(
                   std::back_inserter(outTrackster.vertex_multiplicity()));
       }
       tracksters.resize(1);
+
+      // Find duplicate LCs
+      auto &orig_vtx = outTrackster.vertices();
+      auto vtx_sorted{orig_vtx};
+      std::sort(std::begin(vtx_sorted), std::end(vtx_sorted));
```

Comment:
How many vertices do you expect in this list?
Could the following using `std::unique` be helpful here? https://stackoverflow.com/a/1041970

Reply:
I'm not sure I like having this change bundled with a `Validation` PR.

Reply:
@jpata, we expect O(10) vertices.
By using `std::unique` it would not be possible to know the positions of the duplicated vertices, which is needed to modify the `vertex_multiplicity` list.

@rovere, I would keep it otherwise the rest of the Validation code needs to account for possible duplicate LCs.

Reply:
After chatting with Leo and Felice, I'm fine with this solution.